packages/api/src/utils/auth.js

```diff
@@ -37,19 +39,35 @@ export async function validate(event, { log, db, ucanService }, options) {
   const auth = event.request.headers.get('Authorization') || ''
   const token = magic.utils.parseAuthorizationHeader(auth)
 
-  if (options?.checkUcan && Ucan.isUcan(token)) {
-    const { root, cap } = await ucanService.validateFromCaps(token)
-    const user = await db.getUser(root.audience())
-    if (user) {
-      log.setUser({ id: user.id })
-      return {
-        user: filterDeletedKeys(user),
-        db,
-        ucan: { token, root: root._decoded.payload, cap },
-        type: 'ucan',
+  if (Ucan.isUcan(token)) {
+    if (options?.checkUcan) {
```

Comment:
this should be the other way around right ?

Reply:
No this is correct if it is a UCAN but we don't checkUcans we throw error on line 69 to tell user to go to other route instead